CHANGELOG.yml

```diff
@@ -33,6 +33,11 @@ items:
   - version: 2.19.0
     date: (TBD)
     notes:
+      - type: feature
+        title: Add scheduler name to PODs templates.
+        body: >-
+          A new Helm chart value <code>schedulerName</code> has been added. With this feature, we are 
+          able to define some particular schedulers from Kubernetes to apply some different strategies to allocate resources.
```

Comment:
Suggest wording this so that it speaks about what Telepresence can do instead of what we are able to do.
Also, I think it's important that the comment mentions what pods that are affected.